scripts/check-includes.py

```diff
@@ -15,14 +15,14 @@
 def main(argv):
     argparser = ArgumentParser()
     argparser.add_argument('--generated-includes-dir', action='append',
-                           help='Directory where generated includes are located.')
-    argparser.add_argument('--file', type=open, help='File to check.')
+                           help='Directory where generated includes are located.', required=True)
+    argparser.add_argument('file', type=open, help='File to check.')
     argparser.add_argument('iwyu_args', nargs='*',
-                           help='IWYU arguments, must go after --.')
+                           help='Include-What-You-Use arguments, must go after --.')
     args = argparser.parse_args(argv)
 
     with args.file:
-        iwyu = Popen(['include-what-you-use', '-xc'] + args.iwyu_args + ['/dev/stdin'],
+        iwyu = Popen(['Include-What-You-Use', '-xc'] + args.iwyu_args + ['/dev/stdin'],
```

Comment:
does that work for you  ?? the binary is `include-what-you-use` for me

Reply:
yeah

Reply:
it should not work, please fix it if you want this merged